be/src/exec/schema_scanner/schema_dummy_scanner.cpp

```diff
@@ -19,8 +19,13 @@
 
 namespace doris {
 
+#pragma GCC diagnostic push
```

Comment:
Maybe we can just remove `_s_dummy_columns`?

Reply:
Refined.